### ISSUE TITLE
unicode characters are not rendered

### ISSUE THREAD

**Issue by tessus** [laurent22/joplin#1242]
### Operating system

- Windows
- macOS
- Linux
- Android
- iOS

### Application

- Desktop
- Mobile

---

Joplin 1.0.127 (macOS) and 1.0.237 (Android)

The character  is rendered as

<img width="27" alt="image" src="https://user-images.githubusercontent.com/223439/52985527-38bb9400-33c2-11e9-9154-b6b7ba03a26d.png">

**Comment by CalebJohn** [comment]
Are you sure this is standard unicode? Apple uses a lot of specific unicode symbols that are only available on their products. This character isn't showing up in my browser window (Chrome on android, and brave on linux)

example:
![image](https://user-images.githubusercontent.com/2179547/52991953-0e6ed400-33cc-11e9-839a-329a67bada0a.png)

But if I copy it into joplin (linux) I get 
![image](https://user-images.githubusercontent.com/2179547/52991992-378f6480-33cc-11e9-944a-de758b1faa8a.png)
But it doesn't render on android

**Comment by tessus** [comment]
> Are you sure this is standard unicode?

No, I'm not. But it renders correctly on the previous version of the mobile and desktop apps. So something must have happened since then.
(Also, I can paste and use it in a Linux terminal....)

**Comment by tessus** [comment]
Btw, it is an Apple symbol: <img width="20" alt="image" src="https://user-images.githubusercontent.com/223439/52992572-5c59fb00-3411-11e9-9c57-ecf504176c89.png">

**Comment by tessus** [comment]
https://unicode-table.com/en/F8FF/

**Comment by CalebJohn** [comment]
It looks like this might be a font problem? I know the font in the desktop editor was changed recently, maybe the same happened on android?
Here's a little more info about that [specific character.](https://hea-www.harvard.edu/~fine/OSX/unicode_apple_logo.html)

And here's how it renders in my browser tab bar
![image](https://user-images.githubusercontent.com/2179547/52993632-1598e080-33d2-11e9-9918-06d752d5a133.png)

The page itself
![image](https://user-images.githubusercontent.com/2179547/52993639-1df11b80-33d2-11e9-9fed-c28f7d1dd308.png)

And in gnome-terminal
![image](https://user-images.githubusercontent.com/2179547/52993654-2e08fb00-33d2-11e9-92f9-b8c941aa0711.png)

**Comment by tessus** [comment]
> I know the font in the desktop editor was changed recently, maybe the same happened on android?

The font of the editor seems the same, but the font in the rendering engine has changed. Btw, I like the new font. Looks really nice.

Anyway, you are right, the link I posted above also states that this character is in the Private Use Area block. Thus I shouldn't freakin' use it. It's just something that I've noticed. First it worked, now it doesn't. But just because it worked by chance doesn't mean it's a bug that it doesn't work now.

I wasn't aware that it was from the PUA block when I opened this ticket. Closing...
